style.css

```diff
@@ -231,7 +230,8 @@ ul.card-item {
 
 .items ul {
   display: flex;
-  justify-content: center;
+
+  /* justify-content: center; */
```

Comment:
- [ ] To make your code clean...
Please, clear your project from unused commented codes!

Reply:
done

Reply:
done